Changes

```diff
@@ -1,6 +1,12 @@
 Revision history for Perl extension Net::SSLeay.
 
 ???
+	- Added SSL_in_init() and the related functions for all
```

Comment:
Could you move this item below the next one? It'd be better if changes to minimum version requirements are the first thing people see when they look at the changelog.

Reply:
Will do, I just added the latest to the top but keeping the most important at the top is more useful.